assets/stylesheets/_variables.scss

```diff
@@ -10,6 +10,7 @@ $cr-gray: rgba(151,151,151,1);           // #979797
 $cr-gray-alpha: rgba($cr-gray,.25);      // #979797
 $cr-gray-dark: darken($cr-gray,14);        // #737373
 $cr-gray-darker: darken($cr-gray,29);      // #4D4D4D
+$cr-gray-darkest: darken($cr-gray,35);
```

Comment:
@zarahzachz This naming convention is starting to feel brittle. 

Can you please remove this variable and move the `darken()` function into `_tables.scss`? Considering we're only using this color in this one instance, I don't think this needs to be a global variable (yet).